templates/tools/dockerfile/debian_testing_repo.include

```diff
@@ -0,0 +1,3 @@
+# Add Debian 'testing' repository
+RUN echo 'deb http://ftp.de.debian.org/debian testing main' >> /etc/apt/sources.list
```

Comment:
Do we have to refer to `testing` here?

Reply:
It is the name of the Debian repository. Not the meaning of 'testing' the repository?

Reply:
In `stretch`, to install 3.5, we can use the plain `apt-get`.
To install Python3.6/3.7, we have to add the `testing` repo.

Reply:
Wouldn't `testing` imply the latest `testing` version of Debian? If so, then that'd be destabilizing to our image and akin to be running latest Debian and not Stretch? If not then I am fine with this, otherwise, installing Python yourself and not from the repo sounds best.

Reply:
Debian's stable is fairly glacial when it comes to releasing new software in stable. Doing this will affect any further `apt-get`. So @mehrdada is kind of correct in stating this will affect the rest of the image, but only if there's more installations done after this point.